src/ops/fuelup_toolchain/export.rs

```diff
@@ -0,0 +1,98 @@
+use crate::{
+    commands::toolchain::ExportCommand,
+    toolchain::Toolchain,
+    constants::FUEL_TOOLCHAIN_TOML_FILE,
+    toolchain_override::{self, OverrideCfg, ToolchainCfg, ToolchainOverride},
+};
+use std::collections::HashMap;
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+use semver::Version;
+use anyhow::{bail, Result};
+use component::{self, Components};
+use tracing::info;
+
+pub fn export(command: ExportCommand) -> Result<()> {
+    let ExportCommand { name, force} = command;
+    let path = PathBuf::from("./").join(FUEL_TOOLCHAIN_TOML_FILE);
```

Comment:
We should be smarter about locating the toml file. The user should be able to run this command from any level within the forc project.

You can use `get_fuel_toolchain_toml` for this.